gui-agent-qemu/qubes-gui.c

```diff
@@ -503,6 +499,11 @@ static void qubesgui_message_handler(void *opaque)
                 case MSG_FOCUS:
                 case MSG_EXECUTE:
                     qs->vchan_data_to_discard = qs->hdr.untrusted_len;
```

Comment:
I guess here is the reason https://github.com/QubesOS/qubes-issues/issues/7148 - default case should be one line over.